specs/SystemCursorId

```diff
@@ -0,0 +1,87 @@
+# Background
+
```

Comment:
Note here that this is just for COM & .NET and that a separate WinRT spec will come in the future.